x/incentive/README.md

```diff
@@ -0,0 +1,153 @@
+# Incentive Module
+
+## Abstract
+
+This document specifies the `x/incentive` module of the Umee chain.
+
+The incentive module allows users to `Bond` collateral `uTokens` from the `x/leverage` module, and governance to create and fund `Incentive Programs` which distribute rewards to users with bonded uTokens over time. Users can `Unbond` tokens over a period of time, after which they can be withdrawn. Unbonding time (and reward amounts) depend on which of the three `Unbonding Tiers` uTokens were bonded to.
+
+The incentive module depends on the `x/leverage` module for information about users' bonded collateral, and also requires that the leverage module prevent bonded or currently unbonding collateral from being withdrawn. There are also a few more advanced interactions, such as instantly unbonding collateral when it is liquidated, and registering an `exponent` when a uToken denom is incentivized for the first time.
```

Comment:
This uToken exponent must not change, even if the leverage registry changes an exponent or deletes an asset.

Also, this can be (initially) set to a different value - for example, if we always set the uToken's incentive exponent to +3 from the leverage token's exponent, it will be less vulnerable to rounding & precision problems.

Reply:
Token registry must not change an exponent as well. It will require migration, and will need to be coordinated.